modules/oadp-usecase-backup-odf.adoc

```diff
@@ -0,0 +1,190 @@
+.Prerequisites
+
+. You have installed the {oadp-short} Operator.
+. You have an application with a database running in a separate namespace.
+
+.Procedure
+
+. Create an OBC manifest file to request a NooBaa MCG bucket as shown in the following example:
++
+.Example OBC
+[source,yaml]
+----
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: test-obc # <1>
+  namespace: openshift-adp
+spec:
+  storageClassName: openshift-storage.noobaa.io
+  generateBucketName: test-backup-bucket # <2>
+----
+<1> The name of the object bucket claim.
+<2> The name of the bucket.
+
+. Create the OBC by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <obc_file_name> # <1>
+----
+<1> Specify the file name of the object bucket claim manifest.
+
+. When you create an OBC, {odf-short} creates a `secret` and a `config map` with the same name as the object bucket claim. The `secret` has the bucket credentials, and the `config map` has information to access the bucket. To get the bucket name and bucket host from the generated config map, run the following command:
++
+[source,terminal]
+----
+$ oc extract --to=- cm/test-obc # <1>
+----
+<1> `test-obc` is the name of the OBC.
++
+.Example output
+[source,terminal]
+----
+# BUCKET_NAME
+backup-c20...41fd
+# BUCKET_PORT
+443
+# BUCKET_REGION
+
+# BUCKET_SUBREGION
+
+# BUCKET_HOST
+s3.openshift-storage.svc
+----
+
+. To get the bucket credentials from the generated `secret`, run the following command:
++
+[source,terminal]
+----
+$ oc extract --to=- secret/test-obc # <1>
+----
+<1> `test-obc` is the name of the OBC.
++
+.Example output
+[source,terminal]
+----
+# AWS_ACCESS_KEY_ID
+ebYR....xLNMc
+# AWS_SECRET_ACCESS_KEY
+YXf...+NaCkdyC3QPym
+----
+
+. Get the public URL for the S3 endpoint from the s3 route in the `openshift-storage` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc get route s3 -n openshift-storage
+----
+
+. Create a `cloud-credentials` file with the object bucket credentials as shown in the following command. 
++
+[source,terminal]
+----
+[default]
+aws_access_key_id=<AWS_ACCESS_KEY_ID> # <1>
+aws_secret_access_key=<AWS_SECRET_ACCESS_KEY> # <2>
+----
+<1> Specify the AWS_ACCESS_KEY_ID.
+<2> Specify the AWS_SECRET_ACCESS_KEY.
+
+. Create the `cloud-credentials` secret with the `cloud-credentials` file content as shown in the following command.
++
+[source,terminal]
+----
+$ oc create secret generic \
+  cloud-credentials \
+  -n openshift-adp \
+  --from-file cloud=cloud-credentials
+----
+
+. Configure the DPA as shown in the following example:
++
+.Example DPA
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: oadp-backup
+  namespace: openshift-adp
+spec:
+  configuration:
+    nodeAgent:
+      enable: true
+      uploaderType: kopia
+    velero:
+      defaultPlugins:
+        - aws
+        - openshift
+        - csi
+      defaultSnapshotMoveData: true
```

Comment:
Maybe we should also explain the field as well. All the backups will use data mover option for backing up volumes.

Reply:
Done.